app/views/job_invocations/_form.html.erb

```diff
@@ -81,5 +84,20 @@
     </fieldset>
   </div>
 
+  <!-- modal window -->
```

Comment:
I like the idea of making it nicer, but there's a lot of stuff in here like onclick's.  This is a very simple modal, but they get a lot more complicated when you look at some of the others in Foreman, including additional onclicks, additional sections like footers, default data inside the body, etc.  I think that's maybe why they haven't helper-ized it yet.